tests/storage/sqlite_dos/test_backend.py

```diff
@@ -25,3 +25,15 @@ def test_archive_import(aiida_config, aiida_profile_factory):
         assert QueryBuilder().append(Node).count() == 0
         import_archive(get_archive_file('calcjob/arithmetic.add.aiida'))
         assert QueryBuilder().append(Node).count() > 0
+
+
+def test_backup(aiida_config, aiida_profile_factory, tmp_path, manager):
+    """Test the backup implementation."""
+    with aiida_profile_factory(aiida_config, storage_backend='core.sqlite_dos'):
+        storage = manager.get_profile_storage()
+        storage.backup(str(tmp_path))
+        assert (tmp_path / 'config.json').exists()
+        assert (tmp_path / 'last-backup').is_symlink()
+        dirpath_backup = next(iter(tmp_path.glob('backup_*')))
```

Comment:
Maybe it would be better (both from testing and clarity perspective) to get this directory by resolving the `last-backup` symlink?

Reply:
Fair.  Will change